### ISSUE TITLE
Testers confused by "New Submission" label

### ISSUE THREAD

**Issue by fenekku** [inveniosoftware/invenio-app-rdm#2096]
Package version (if known): v11

*This is more of a feedback / possible alternative from our group of testers*

Testers were confused by the label "New Submission". Since this label is kept even when the record submission is accepted/rejected or even after a couple of days, testers didn't understand what it meant.

I *think* it means "this request is for a record to become part of the community" as opposed to being a request for quota increase / a user to become part of the community. With that in mind, testers found "Record submission" / "Record inclusion" / "Upload inclusion" more clear. The crux of the problem was the term "new": what was new? when was something not new? "new" implied recent or different than pre-existing other requests which isn't the case. 

![image](https://user-images.githubusercontent.com/1932651/219485758-f3fb869d-edf0-472a-b311-e2b5ab8ada0b.png)

**Comment by ntarocco** [comment]
Thanks a lot for the feedback, totally agree :) The situation will get even more complicated with InvenioRDM v12: we are introducing a new type of request, technically called `community-inclusion`, that allows including an already published record to multiple communities.
The challenge is to find good names for our users, that are more understandable than the technical names.
The label needs to be changed in any case, and the `New` part does not make much sense.

After some internal brainstorming, below some proposals:
1. Use `community inclusion` for the new feature and rename the current `community submission` to one of:
- Publish review
- Peer review
- Upload review
- First review
- First publish
- Initial review
- Draft review

 ![Screenshot 2023-03-08 at 15 25 45](https://user-images.githubusercontent.com/2089455/223751288-6b1728cd-c65c-4da4-9a39-26c9ee962851.jpg)

2. Name both requests `community inclusion`, but add **`Draft` or similar** to the community-submission and change the UI label by splitting it in 2 labels:

![Screenshot 2023-03-08 at 15 24 46](https://user-images.githubusercontent.com/2089455/223751274-5b20718a-6f83-4b68-8175-e2811625acf5.jpg)

3. Name both requests `community inclusion`, but have very long names. This is just an example:
![Screenshot 2023-03-08 at 15 26 44](https://user-images.githubusercontent.com/2089455/223751297-8a2fcb11-6005-408b-b027-6d3e48f6bcc8.jpg)

We will continue our discussion internally, it would be nice to have your feedback too.

**Comment by fenekku** [comment]
Got it, it's indeed a community inclusion :+1: . I think screenshot 2 and 3 are flipped. After flip, 2 is probably closer to what makes sense to me, but I will share this with colleagues for their feedback. Thanks!

**Comment by ntarocco** [comment]
> Got it, it's indeed a community inclusion :+1: . I think screenshot 2 and 3 are flipped. After flip, 2 is probably closer to what makes sense to me, but I will share this with colleagues for their feedback. Thanks!  

Yes you are right, sorry about that. We have discussed this today, we will prepare some new mock ups, taking into account other future request types.

**Comment by fenekku** [comment]
After prodding about this, the feedback from our stakeholders revolved around a couple of things:

- It would be clearer if the labels paralleled each other: 

label: Member invitation 
-- TO --> 
label: **Member inclusion or Member community inclusion**
title (for invitation): Your invitation to join "Open public community" 
title (for join request): Your request to join "Open public community"

label: Community inclusion 
-- TO --> 
label: **Record inclusion or Record community inclusion**

These labels always put *what* is included in the label.

- There is still confusion about why in the community dashboard, member invitations (a kind of request) are not in the requests tab. It breaks our stakeholders' expectations of this being the same as for records and the same as in a user's dashboard.

- With respect to the concept of a new record (draft) versus a published record being displayed on the label or next to it, it didn't matter so much as long as both concepts were labelled. That is, if it's a published record being included, something to the effect that it is a published record should be present (and if it's a draft, something to the effect of it being a draft should be present). Draft or Published should not be omitted in one of the cases.

I'll relay other feedback as they get to use the system more.
